ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/policy-maps.md

```diff
@@ -84,6 +84,13 @@ policy-map type pbr PM_PBR_BREAKOUT
 | CM_REPLICATION_LD | dscp | af11 |
 | CM_REPLICATION_LD | cos | 4 |
 
+##### PM_REPLICATION_LD3
+
+| class | Set | Value |
```

Comment:
Please add the police stuff to this table. Try to keep it as one or two columns.

Reply:
added